File: problem_6/python/problem6.py
#!/usr/bin/python
"""Project Euler solution using generators."""

def generator():
    for x in range(1,100+1):
        yield (x,x*x)
sum1 = 0
sum2 = 0

for i in generator():
    sum1 += i[0]
    sum2 += i[-1]

print(sum1 * sum1 - sum2)

File: problem_6/java/problem6.java
public class problem6
{
    public static void main( String[] args)
    {
      int sum1 = 0;
      int sum2 = 0;
      
      for( int i = 0; i < 100; i++)
      {
        sum1 += i;
        sum2 += i ^ 2;
      }

      System.out.println(sum1 ^ 2 - sum2);
    }
}
